test/sensor.js

```diff
@@ -8,7 +8,43 @@ var MockFirmata = require("./util/mock-firmata"),
     io: new MockFirmata(),
     debug: false,
     repl: false
-  });
+  }),
+  // TODO: query: does node.js / es6 have an object clone / deepcopy method?
```

Comment:
That was a note to myself, to be looked up.  Had not gotten around / back to it yet.  Answered now, so I do not need to do the research. :)

I had the code to do it myself, but thought new version might have it built in.  Much prefer to use 'native' code when it is available.  I was writing that section when I had no online access.